raisinbread/instruments/NDB_BVL_Instrument_EEG_Acquisition.class.inc

```diff
@@ -0,0 +1,993 @@
+<?php
```

Comment:
I don't see why this isn't a LINST instrument. It doesn't seem to do anything other than add basic field types and use XIN rules, both of which are supported by LINST.

Reply:
Agreed.